src/Client/WebSocketClient.php

```diff
@@ -15,10 +15,10 @@
 use Nekland\Woketo\Message\MessageHandlerInterface;
 use Nekland\Woketo\Rfc6455\FrameFactory;
 use Nekland\Woketo\Rfc6455\MessageFactory;
-use Nekland\Woketo\Rfc6455\MessageHandler\CloseFrameHandler;
-use Nekland\Woketo\Rfc6455\MessageHandler\PingFrameHandler;
-use Nekland\Woketo\Rfc6455\MessageHandler\RsvCheckFrameHandler;
-use Nekland\Woketo\Rfc6455\MessageHandler\WrongOpcodeHandler;
+use Nekland\Woketo\Rfc6455\FrameHandler\CloseFrameHandler;
+use Nekland\Woketo\Rfc6455\FrameHandler\PingFrameHandler;
+use Nekland\Woketo\Rfc6455\FrameHandler\RsvCheckFrameHandler;
+use Nekland\Woketo\Rfc6455\FrameHandler\WrongOpcodeHandler;

```

Comment:
1- I's useless to repeat the directory name 
2- If you continue with this convention, you can end up with classes with a very long name. 
3- You did not answer my question why wrongcode does not use the same convention as the other classes
5 - If you do not want to follow my recommendations, I do not care... really really... But I note that you refuse the discussion
4 - What do the others core team's members think ? @Nekland/core